conmon/conmon.c

```diff
@@ -395,6 +396,9 @@ static int write_k8s_log(int fd, stdpipe_t pipe, const char *buf, ssize_t buflen
 		if ((opt_log_size_max > 0) && (bytes_written + bytes_to_be_written) > opt_log_size_max) {
 			bytes_written = 0;
 
+			if (writev_buffer_flush(fd, &bufv) < 0) {
+				nwarn("failed to flush buffer to log");
```

Comment:
same comment as master PR, should we exit here instead of just logging a warning? we'll get a corrupted log file on reopen otherwise